src/AWS.Logger.AspNetCore/AWSLoggerConfigSection.cs

```diff
@@ -81,6 +146,10 @@ public AWSLoggerConfigSection(IConfiguration loggerConfigSection)
             {
                 Config.Profile = loggerConfigSection[PROFILE];
             }
+            if (loggerConfigSection[PROFILE_LOCATION] != null)
```

Comment:
Could we try https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.configurationbinder.bind?view=aspnetcore-2.2#Microsoft_Extensions_Configuration_ConfigurationBinder_Bind_Microsoft_Extensions_Configuration_IConfiguration_System_Object_ ?

Reply:
Not all of the values go on to Properties of `AWSLoggerConfigSection`.  About half of the values go to the underlying `AWSLoggerConfig` object. The Bind operation is not going to handle that.